.github/workflows/trunk.yml

```
@@ -51,8 +51,22 @@ jobs:
           echo "matrix=$matrixStringifiedObject" >> $GITHUB_OUTPUT
         env:
           PROJECT_ROOT: .
+      - name: Get Latest Github Relaase
+        id: get_latest_release
+        uses: actions/github-script@v4
```

actionlint (1.7.4)

the runner of "actions/github-script@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)